src/main/java/filter/expression/Qualifier.java

```diff
@@ -231,6 +236,9 @@ public boolean isSatisfiedBy(IModel model, TurboIssue issue, MetaQualifierInfo i
         // The empty qualifier is satisfied by anything
         if (isEmptyQualifier()) return true;
 
+        // The false qualifier is satisfied by nothing
+        if (isFalseQualifier()) return false;
```

Comment:
I implemented it as opposite of `Empty` qualifier. Its name would be `false` and its content  would be empty.